ironpdf/src/main/java/com/ironsoftware/ironpdf/PdfDocument.java

```diff
@@ -1762,14 +1776,16 @@ public final String extractTextFromPage(PageSelection pageSelection) {
     /**
      * Replace the specified old text with new text on a given page.
      *
-     * @param pageSelection The selected page index(es).
-     * @param oldText       Old text to remove
-     * @param newText       New text to add
+     * @param pageSelection     The selected page index(es).
+     * @param oldText           Old text to remove
+     * @param newText           New text to add
+     * @param fontName          The font to use for the new text
+     * @param customFontSize    The font size to use for the new text
      */
-    public final void replaceText(PageSelection pageSelection, String oldText, String newText) {
+    public final void replaceText(PageSelection pageSelection, String oldText, String newText, String fontName, float fontSize) {
```

Comment:
This  fontName and  fontSize is optional right? so please add overload instead.

Reply:
I'm right. If I dont pass any font name to gRPC then it would throw exception saying that the font name '' is not valid standard font. So I'm just checking for null and assign TimesNewRoman to it.

Reply:
I see, IronPdfEngine does not handle this correctly.
your solution is fine for now. (next version the api will change again)